nx-dev/feature-package-schema-viewer/src/lib/parameter-view.tsx

```diff
@@ -61,16 +61,20 @@ export const ParameterView = (props: {
     </div>
 
     <div className="prose prose-slate dark:prose-invert">
-      {renderMarkdown(props.description, {
-        filePath: '',
-      })}
+      {
+        renderMarkdown(props.description, {
+          filePath: '',
+        }).node
+      }
     </div>
 
-    {props.deprecated && !!props.schema['x-deprecated'] && (
+    {props.deprecated && !!(props.schema as any)['x-deprecated'] && (
```

Comment:
Should be able to use `props.schema['x-deprecated']` without casting right? The value is unknown but it shouldn't matter here.

Reply:
It actually matter since this is not `unknown` but default to `implicit any` which makes typescript scream. The whole typing of this view is a bit outside of the PR right now since to properly resolve it, few files are involved (so Typescript infers the right type (from `NxSchema` not default `JsonSchema1` interface)).